File: api/Controllers/Models/Entities/User_Role_Permission.cs
using System;
using System.ComponentModel.DataAnnotations;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace IPKP___API.Controllers.Models.Entities
{
    public class User_Role_Permission
    {
        [Key]
        public Guid User_Role_Permission_ID { get; set; }

        public int User_Role_ID { get; set; }
        public User_Role User_Role { get; set; }

        public int Permission_ID { get; set; }
        public Permission Permission { get; set; }
    }
}

File: admin/src/app/Models/userrolepermission.ts
import { UserRole } from "./userrole"
import { Permission } from "./permission"
export class UserRolePermission{
    user_Role_Permission_ID!: string
    
    user_Role_ID!: string
    user_Role!: UserRole

    permission_ID!: string 
    permission!: Permission
}